src/client/pages/signIn/SignIn.jsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import { useState } from 'react';
 import '../../styles/signIn.css'
+import Header from '../../components/Header'
+
+const URL = process.env.REACT_APP_API_URL;
```

Comment:
I don't know how to do config files so I'll leave it for now

Reply:
A config file is simply a file that contains values to use as settings for things. It could be a file that contains things like:

```js
export const APP_NAME = 'My App';
export const FAVOURITE_COLOUR = 'Purple';
```

I'd recommend doing this asap, particularly because you've said you don't know how to do it :)